Scripts/DCS-BIOS/LocalCompile.lua

```diff
@@ -0,0 +1,18 @@
+-- This can be used to run the DCS-BIOS JSON generation locally without
+-- starting a DCS mission
+-- DCS-BIOS wiki have information on how to install the lua interpreter. No need to install
+-- additional lua modules such as socket etc.
+-- Execute => lua LocalCompile-lua
+
+local dcs_variant = "DCS"
+DCS_USER_DIR_ROOT = os.getenv ("USERPROFILE").."\\Saved Games\\"..dcs_variant.."\\"
+DCS_INSTALLATION_ROOT = [[D:\DCS World]]
```

Comment:
Not everyone will have dcs installed here. If you want to do it this way, you might try using a registry key. An alternative would be to include the json.lua here so that the code isn't dependent on being run on the same machine as dcs

Reply:
It was meant that each user set it's own value there

Reply:
OK

Reply:
Honestly I think just copying over the json.lua file would be best, because then this can be run even when dcs isn't installed on the machine (like in the unit test runners)